.github/CODEOWNERS

```diff
@@ -427,7 +427,7 @@
 /packages/grafana-ui/src/components/PluginSignatureBadge/ @grafana/plugins-platform-frontend
 /packages/grafana-ui/src/components/Sparkline/ @grafana/grafana-frontend-platform @grafana/app-o11y-visualizations
 /packages/grafana-ui/src/components/Table/ @grafana/dataviz-squad
-/packages/grafana-ui/src/components/Table/SparklineCell.tsx @grafana/dataviz-squad @grafana/app-o11y-visualizations
+/packages/grafana-ui/src/components/Table/Cells/SparklineCell.tsx @grafana/dataviz-squad @grafana/app-o11y-visualizations
```

Comment:
is this change relevant to the PR?

Reply:
I got an CODEOWNERS build error because of a missing file, maybe I can ignore that build error?

Reply:
Fixed by someone else in https://github.com/grafana/grafana/commit/db50c3c5fb9516c19c4e8cfa3ade53bafde9dbaf